kiwi_lint/__init__.py

```diff
@@ -26,5 +27,6 @@ def register(linter):
     linter.register_checker(ObjectsUpdateChecker(linter))
     linter.register_checker(TagsChecker(linter))
     linter.register_checker(EmptyModuleChecker(linter))
+    linter.register_checker(EmptyClassChecker(linter))
```

Comment:
Can you rebase and move this class into `empty.py`. That is a more natural place for it.